tests/test_selection/test_eliminate_feature_by_feature_importance.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
```

Comment:
can we change the name of the test to test_recursive_feature_elimination.py? so that it mirrors both the name of the class and the python script if you renamed it?